### ISSUE TITLE
Tooltips should be keyboard accessible

### ISSUE THREAD

**Issue by ggeisler** [pod4lib/aggregator#846]
It looks like our tooltips are not keyboard accessible. For example, if I'm on this page (as a POD admin):

https://pod.stanford.edu/organizations/stanford

and begin tabbing, after getting to the organization header I tab from "Manage organization" straight to "Manage streams" and therefore bypass the "Default" badge with the tooltip showing when this stream was the default stream.

The [Bootstrap tooltip page](https://getbootstrap.com/docs/5.0/components/tooltips/#markup) talks a little bit about accessibility, although I'm not sure how helpful it is for our particular use cases. But maybe some investigation will reveal a way to modify things to make the tooltips accessible via a keyboard.
